index.html

```diff
@@ -328,6 +434,114 @@ <h3>
+                  empty dictionary. However, passing a dictionary like `{text:
+                  "text", files: []}` is fine, as `files` is just ignored.
+                </p>
+              </li>
+              <li>If the implementation does not support file sharing, return
+              false.
+              </li>
+              <li>If the user agent believes sharing any of the files in
+              `files` would result in a potentially hostile share, return
+              false.
+                <div class="issue" data-number="127"></div>
+              </li>
+            </ol>
+          </li>
+          <li>If |data|'s url member is present:
+            <ol>
+              <li>Let |url:URL| be the result of running the [=URL parser=] on
+              |data|'s url, with |base|, and no encoding override.
+              </li>
+              <li>If |url| is failure, return false.
+              </li>
+              <li>If |url|'s [=URL/scheme=] is not "http" or "https", return
+              false.
+              </li>
+            </ol>
+          </li>
+          <li>Return true.
+          </li>
+        </ol>
+      </section>
+      <section data-dfn-for="CanShareQuery">
+        <h3>
+          `CanShareQuery` dictionary
+        </h3>
+        <pre class="idl">
+            dictionary CanShareQuery {
+              DOMString type;
+            };
+          </pre>
+        <section>
+          <h4>
+            `type` member
+          </h4>
+          <p>
+            The <dfn>type</dfn> member is a [=MIME Type=] that the developer
+            would like to check support for.
+          </p>
+        </section>
+      </section>
+      <section data-dfn-for="SupportedShareMember">
+        <h3>
+          Supported share members
+        </h3>
+        <p>
+          The <dfn>supported share members</dfn> provide a means for developers
+          to check if particular sharing capabilities are supported by the user
+          agent. They are represented by the following string literals:
+        </p>
+        <dl>
+          <dt>
+            "<dfn>files</dfn>"
+          </dt>
+          <dd>
+            Represents support for the {{ShareData/files}} member.
+          </dd>
+          <dt>
+            "<dfn>url</dfn>"
+          </dt>
+          <dd>
+            Represents support for the {{ShareData/url}} member.
+          </dd>
+          <dt>
+            "<dfn>text</dfn>"
+          </dt>
+          <dd>
+            Represents support for the {{ShareData/text}} member.
+          </dd>
+        </dl>
+        <p class="Note" title="Why not an enum?">
+          The choice to use string literals instead of an [=enumeration=] is
+          deliberate. Doing so allows implementers to gracefully support future
+          {{ShareData}} member values, while also giving the user agent the
+          ability to disable sharing those members if needed.
```

Comment:
Should this mention https://github.com/heycam/webidl/issues/893 or https://github.com/heycam/webidl/issues/491?

Reply:
No. If something changes in WebIDL, then it should be adopted. heycam/webidl#491 should be closed anyway - it's no longer relevant.